core/src/main/scala/matryoshka/package.scala

```diff
@@ -316,24 +343,86 @@ package object matryoshka extends CofreeInstances with FreeInstances {
 
   /** Count the instinces of `form` in the structure.
     */
-  def count[T[_[_]]: Recursive, F[_]: Functor: Foldable](form: T[F]): F[(T[F], Int)] => Int =
+  def count[T[_[_]]: Recursive, F[_]: Functor: Foldable](form: T[F]):
+      F[(T[F], Int)] => Int =
     e => e.foldRight(if (e ∘ (_._1) == form.project) 1 else 0)(_._2 + _)
 
-  implicit def ToIdOps[A](a: A): IdOps[A] = new IdOps[A](a)
+  /** The number of nodes in this structure.
+    */
+  def size[F[_]: Foldable]: F[Int] => Int = _.foldRight(1)(_ + _)
+
+  /** The largest number of hops from a node to a leaf.
+    */
+  def height[F[_]: Foldable]: F[Int] => Int = _.foldRight(0)(_ max _)
+
+  /** Combines a tuple of zippable functors.
+    */
+  def zipTuple[T[_[_]]: Recursive, F[_]: Functor: Zip]:
+      Coalgebra[F, (T[F], T[F])] =
+    p => Zip[F].zip[T[F], T[F]](p._1.project, p._2.project)
+
+  /** Aligns “These” into a single structure, short-circuting when we hit a
+    * “This” or “That”.
+    */
+  def alignThese[T[_[_]]: Recursive, F[_]: Align]:
+      ElgotCoalgebra[T[F] \/ ?, F, T[F] \&/ T[F]] =
+    _.fold(_.left, _.left, (a, b) => a.project.align(b.project).right)
 
-  implicit def ToCorecursiveOps[T[_[_]]: Corecursive, F[_]](f: F[T[F]]):
-      CorecursiveOps[T, F] =
-    new CorecursiveOps[T, F](f)
+  /** Merges a tuple of functors, if possible.
+    */
+  def mergeTuple[T[_[_]]: Recursive, F[_]: Functor: Merge]:
+      CoalgebraM[Option, F, (T[F], T[F])] =
+    p => Merge[F].merge[T[F], T[F]](p._1.project, p._2.project)
+
+  /** Generates an infinite sequence from two seed values.
+    */
+  def binarySequence[A](relation: (A, A) => A): Coalgebra[(A, ?), (A, A)] = {
+    case (a1, a2) =>
+      val c = relation(a1, a2)
+      (c, (a2, c))
+  }
+
+  /** Converts a fixed-point structure into a generic Tree.
+    * One use of this is using `.cata(toTree).drawTree` rather than `.show` to
+    * get a pretty-printed tree.
+    */
+  def toTree[F[_]: Functor: Foldable]: Algebra[F, Tree[F[Unit]]] =
+    x => Tree.Node(x.void, x.toStream)
```

Comment:
I'm not convinced we would actually use this because the `()`s uglify the result with any sensible Show, but it's clever and that's enough.